testdata/go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/ckaznocha/intrange/testdata
 
-go 1.21
+go 1.22
```

Comment:
Please revert this line. To be compatible with golangci-lint it needs to support go1.21.

Reply:
Didn't know that. Done.

Reply:
Oh, this was the test go.mod, that probably would have been fine. Sorry, that's what I get for doing pr reviews on my phone.